pyuvdata/uvdata/mwa_corr_fits.py

```diff
@@ -345,6 +774,16 @@ def read_mwa_corr_fits(
                         file_dict["data"] = [file]
                     else:
                         file_dict["data"].append(file)
+
+                    # save bscale keyword
+                    if bscale is None:
```

Comment:
It looks to me like `bscale` is hard-coded to `None`, so I'm not sure this if statement is required.

Reply:
This was originally a check to see if `bscale` had been assigned a new value already, so that it didn't get reassigned each time a file was opened. Now it is effectively a check to see if the bscale has been stored in an extra keyword, again to prevent reassignment with each file. It would probably make sense now to do the keyword check directly instead of depending on `bscale`.

Reply:
I see, it's inside the file loop. So it's just being set by the first file?

Reply:
yes